waspc/src/Wasp/Wasp/Db.hs

```diff
@@ -4,8 +4,8 @@ module Wasp.Wasp.Db
   )
 where
 
-data Db = Db
-  { _system :: !DbSystem
+newtype Db = Db
```

Comment:
`data`

Reply:
If you dislike changing to `newtype` what will soon be `data` again, I followed hlint's suggestions one warning type at a time and could revert the commit that converts `newtype` to `data`.

Reply:
I reverted the `data` to `newtype` changes.

Reply:
awesome, thanks!